diesel_cli/src/main.rs

```diff
@@ -191,22 +221,23 @@ fn redo_latest_migration<Conn>(conn: &Conn) where
     }).unwrap_or_else(handle_error);
 }
 
-fn handle_error<E: Error>(error: E) {
+fn handle_error<E: Error, T>(error: E) -> T {
```

Comment:
`src/main.rs:175:71: 175:85 error: the trait `core::ops::FnOnce<(database_error::DatabaseError,)>` is not implemented for the type `fn(_) -> ! {handle_error}` [E0277]`